docs/timelines/creating-a-timeline.md

```diff
@@ -0,0 +1,99 @@
+
+Trigger with multiple points during Live Record:
+
+![Timeline before auto simplify](/docs/images/Timeline-Live-Record.png)
+
+The same trigger after Auto Simplify:
+
+![Timeline after auto simplify](/docs/images/Timeline-Live-Record-Simplified.png)
+
+- When you select a trigger (by clicking on it), there are also context menu options in the \{Tools\} context menu to \{Simplify Selected Triggers\} and \{Smooth Selected Triggers\}. The Smooth option will smoothly fade between selected triggers using the fade time of the following trigger.
+
+Adding manual triggers
+-----------------
+
+If you are programming a show where you have been given a list of timecodes for each cue point, you can enter the triggers manually at those specific times.
+
+1.  Press \<Record\>, \[Timeline\], then the **Select button** of the fader where you want to store the timeline. The Timeline View window will automatically open.
+
+2.  Press the \{+\} button at the bottom of the timeline tracks.
+
+    ![Timeline add trigger window](/docs/images/Timeline-Add-Item.png)
+
+3. Select the type of trigger you want to add.
+    - **New Playback** will create a playback from the current programmer contents and trigger it at 100% for 2 seconds then off.
+    - **Existing Playback** lets you select a playback you've programmed previously, and inserts it at 100% for 2 seconds then off.
+    - **Set Level** inserts a trigger to set an existing playback to a specified level.
+    - See the [Triggers](../timelines.md#triggers) section for details of the other types of trigger.
+
+4. If you are using an existing playback, press the **Select** button of the playback you want to fire. (You can skip step 3 and just select a playback straight after pressing the \{+\} button, Titan will behave as if you pressed **Existing Playback**.)
+
+5. Set the time you want the trigger to fire at. There are several ways to do this:
+    - Type in the time using the \[Reference at hh:mm:ss.ttt \] softkey and press \<Enter\>. Use the \<.\> button as a separator, for example typing "1 . 05" will enter 00:01:05:000 or "2 . 2 . 20" will enter 02:02:20.00.
+    - Use the current Live Time (blue cursor position) using the \[Reference at Live Time\] softkey.
+    - Tap on the Timeline View on the screen at the place where you want the trigger (you can adjust this later).
+
+6. Repeat from Step 2 to add more triggers. 
+
+
+- The [Table View](../timelines/running-and-editing-timelines.md#table-view) can be easier to work with than the graphical timeline when entering timecodes manually as it lets you see
+the numbers you have entered.
+
+### Importing markers
```

Comment:
I think we should probably include some steps or at least a link to how to export from reaper. I'll try and write some steps or find a link for that for you.

Reply:
I don't have Reaper here and have not ever used it so some steps would be useful. There is a guide (for some other lighting software) at https://www.manula.com/manuals/avld/mla/131/en/topic/importing-timecode-exporting-from-reaper which I can use as instruction if it is still accurate

Reply:
Yep, those are the correct steps.

Sebastian has also written some steps on Avolites.de. It's steps 1-4 https://www.avolites.de/wiki/external:reaper_csv_import to get the file.

You should then be able to place it in the `Documents\Titan` folder on PC Suite or on a USB stick at `USB\` or `USB\Titan`